### ISSUE TITLE
Optimization for FURY(fiji) on allium/phi2/x22i

### ISSUE THREAD

**Issue by Stargazer77** [fancyIX/sgminer-phi2-branch#127]
Please optimize allium/phi2/x22i  for FURY(fiji)(HBM) ! There are a lot of them in mining(we have a lot of too) and they are better than RX580 in many algorithms, but only after working as developers.  They have HBM memory and very like on Vega! We waiting this update from you! Thank you!

**Comment by Stargazer77** [comment]
Yes, I know what they supported, I told about optimization work for this
GPU, because 2.2Mh/s on allium(less on 45% then RX580, and on other algo
with this miner  results the same) its just funny! FIJI  faster on all algo
then RX580 and must be faster on this algos too!  And I am test rename the
prebuilt a two bin files: "alliumBaffingw256l4" and
"alliumEllesmeregw256l4", and with "alliumBaffingw256l4" this works a
little faster then "alliumEllesmeregw256l4"!  FURY have a hbm memory and
should work something like VEGA!! What you think about this?

ср, 21 нояб. 2018 г. в 04:26, fancyIX <notifications@github.com>:

**Comment by fancyIX** [comment]
@Stargazer77 Can you read again what I said? I mean Fiji is already optimized. You just need to rename a file to make the optimized binary file work.

**Comment by fancyIX** [comment]
@Stargazer77 What is the speed with your renamed file? only 2.2mhs? If so, there should be some problem. Fury seems like Vega with lower clock frequency. Interesting to see why its slower than expected. HBM should not help much since the current implementation is not memory bound.

**Comment by Stargazer77** [comment]
Because I do not think this result is an optimization. Binary files I
changed (from 1mh/s it became 2.2mh/s), read my message carefully again. I
think you need to work with this model(FURY/NANO) and see what the problem
is. Can we expect improvements to this GPU from you?

ср, 21 нояб. 2018 г. в 05:09, fancyIX <notifications@github.com>:

**Comment by Stargazer77** [comment]
Maybe someone lives next to you and will be able  to gives this GPU for
tests for a while? I would love to, but I live far away. And Postal
services are more expensive than the Nano / Fury video card.

ср, 21 нояб. 2018 г. в 05:26, fancyIX <notifications@github.com>:

**Comment by fancyIX** [comment]
@Stargazer77 It's still expensive even for second hand. If someone can lend it to me I will spend some time to do some testing on it.

**Comment by fancyIX** [comment]
Won't fix because IP theft is happening (not to this miner though) in some closed source miner. It’s a huge effort to write GCN asm code and build optimized bin kernels including GCN code. If someone can just use your bin files in closed source miner without giving you any credit, what is the point of those effect building those bin kernels...
